HelloMixpanel/HelloMixpanel_tvOS/ViewController.m

```diff
@@ -22,7 +22,7 @@ - (IBAction)setPeopleProperties:(id)sender {
     
     // Mixpanel People requires that you explicitly set a distinct ID for the current user. In this case,
     // we're using the automatically generated distinct ID from event tracking.
-    [mixpanel identify:mixpanel.distinctId];
+   // [mixpanel identify:mixpanel.distinctId];
```

Comment:
why'd we comment this out?

Reply:
that was for a temp test, will revert it back.